### PR TITLE
add comparitech scraper

### PR DESCRIPTION
**Pull request by gabrieltorresgamez**

### CODE REVIEW

**Comment by gabrieltorresgamez** [comment]
oh i forgot tests, dont approve yet

**Comment by gabrieltorresgamez** [comment]
ok now

File: tests/test_scraper_comparitech.py
Comment (can020202) [inline tests/test_scraper_comparitech.py:1]: there is still GoogleResultsScraper in Docstrings

**Review by can020202 — CHANGES_REQUESTED** [review]
in test_scraper_comparitech.py there is still GoogleResultsScraper in Docstrings, rest looks good

**Comment by gabrieltorresgamez** [comment]
massive spot, is fixed now :)

**Review by can020202 — APPROVED** [review]
looks good

**Pull request merged**